docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,2465 @@
+{
```

Comment:
<p>should we show how? e.g. </p><p> </p><p><code>sc.pl.matrixplot(adata, groupby="cell_type", var_names=["CD8A", "CD4", "KLRB1"], <strong>layer="counts_per_million"</strong>)</code> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>